tests/BuildScriptGenerator.Tests/Python/PythonPlatformTests.cs

```diff
@@ -235,67 +228,19 @@ public void Detect_ReturnsVersionFromOptions_EvenIfRuntimeTextFileHasVersion()
             Assert.Equal(expectedVersion, result.PlatformVersion);
         }
 
-        [Fact]
-        public void Detect_ReturnsResult_WhenOnlyMajorAndMinorVersion_AreSpecifiedInRuntimeTxtFile()
```

Comment:
same here

Reply:
Sure, in general I agree with the concern regarding lost coverage. In this test instead of now parsing the runtime.txt file and then resolving the version, we are assuming that the PythonDetector gave us a detected version and we are getting the max satisfying version. And for this we have a test in this PR called `Detect_ReturnsResult_WhenOnlyPartialVersionIsProvidedByDetector`